lib/model/security/CliLoginModel.class.php

```diff
@@ -119,9 +119,6 @@ public function login($user, $pass_plain, $evtWachten = true, RememberLogin $rem
 		// Subject assignment:
 		self::$uid = $account->uid;
 
-		// Permissions change: delete old session
-		session_regenerate_id(true);
```

Comment:
Een CLI sessie begint altijd vanaf 0, er is hier geen oude sessie om te verwijderen. Dit is overgebleven uit `LoginModel`. Veroorzaakte warnings.